src/functions.c

```diff
@@ -147,6 +170,14 @@ static void engineLibraryDispose(dict *d, void *obj) {
 void functionsLibCtxClear(functionsLibCtx *lib_ctx) {
     dictEmpty(lib_ctx->functions, NULL);
     dictEmpty(lib_ctx->libraries, NULL);
+    dictIterator *iter = dictGetIterator(lib_ctx->engines_stats);
+    dictEntry *entry = NULL;
+    while ((entry = dictNext(iter))) {
+        functionsLibEngineStats *stats = dictGetVal(entry);
+        stats->n_functions = 0;
+        stats->n_lib = 0;
+    }
+    dictReleaseIterator(iter);
```

Comment:
Why not `dictEmpty(lib_ctx->engines_stats)`?

Reply:
@MeirShpilraien I'm back..why not move `n_lib` and `n_functions` to `engineInfo`?

Reply:
@sundb engineInfo is singleton, for example, you might load libraries during diskless replication with backup option that you do not want to count until there is that actual swapdb. Per engine stats are linked with the libraries_ctx and relevant only to current libraries_ctx.